### PR TITLE
🎨 Update CSS classes applied to math nodes to match `remark-math`

### PR DESCRIPTION
**Pull request by sglyon**
change `math block` to `math-display` and `math inline` to `math-inline`

ref #859

### CODE REVIEW

**Comment by sglyon** [comment]
I'm working on fixing some tests right now! Will ping when ready!

**Comment by sglyon** [comment]
@rowanc1 ok I think we are about ready to go.

I added some very simple tests to `myst-to-html`

It seems like `myst-parser` gets some tests based on the examples in the `myst-spec` package. I make a related PR over there to update the class names: https://github.com/executablebooks/myst-spec/pull/63

So, if we are able to merge/release myst-spec and then update the dependency over here the tests should pass ;)

**Comment by rowanc1** [comment]
Thanks so much for this work @sglyon.

A question @agoose77 had this morning was ensuring that we don't need to also add `$$` or `\[` or other text delimiters into the output? Do you have knowledge of if that will be desirable/necessary in the remark/rehype pipelines?

File: packages/myst-to-html/src/schema.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ const math: Handler = (h, node) => {
 };
 
 const inlineMath: Handler = (h, node) => {
-  return h(node, 'span', { class: 'math inline' }, [
+  return h(node, 'span', { class: 'math-inline' }, [
```
Comment (rowanc1) [inline packages/myst-to-html/src/schema.ts:68]: So many changes for a `-`!! 🚀  Thanks again @sglyon for your help on this.

**Comment by sglyon** [comment]
Hey @rowanc1 I don't think it will be necessary because of how the ast looks, but I don't know for sure... 

From what I can tell by the tests over there I thiknk we will be good without adding additional characters: https://github.com/remarkjs/remark-math/blob/e99b9d088709d743adf6a43551fd416d7e0014ed/packages/rehype-katex/test.js#L27-L47

**Comment by rowanc1** [comment]
Fantastic! Thank you, will get this in shortly!! (Away from my keyboard now..)

**Comment by rowanc1** [comment]
Thanks again @sglyon for your help in tracking these down and improving them in various repos!

**Comment by sglyon** [comment]
Awesome!! looking forward to a release and I'll test it out in my app ;)

**Comment by sglyon** [comment]
@rowanc1 could we get a new build out for `myst-to-html`? Then I can start using 👏

**Comment by rowanc1** [comment]
We are planning on releasing later today!

**Comment by rowanc1** [comment]
Version 1.0.22 should now be out! Thanks again for your help.

**Comment by sglyon** [comment]
Amazing — thank you!

On January 29, 2024, Spencer Lyon ***@***.***> wrote:
> Version 1.0.22 should now be out! Thanks again for your help.
>

—
Reply to this email directly, view it on GitHub
<https://github.com/executablebooks/mystmd/pull/862#issuecomment-
1915822159>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AAHUTXHVTJR2ROYGG7CGINDYRA4ZDAVCNFSM6AAAAABCFDFIPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMJVHAZDEMJVHE>.
You are receiving this because you were mentioned.Message ID:
***@***.***>

**Pull request merged**